src/components/Popups/PopupDrawingToolbar.tsx

```diff
@@ -57,9 +59,25 @@ const PopupDrawingToolbar = ({
     reference,
 }: Props): JSX.Element => {
     const intl = useIntl();
+    const popupRef = React.useRef<PopupBase>(null);
+    const prevReference = usePrevious(reference.getBoundingClientRect());
+
+    React.useEffect(() => {
+        const { current: popup } = popupRef;
+
+        if (popup?.popper && prevReference) {
+            const { height: prevHeight, left: prevLeft, top: prevTop, width: prevWidth } = prevReference;
+            const { height, left, top, width } = reference.getBoundingClientRect();
+
+            if (height !== prevHeight || left !== prevLeft || top !== prevTop || width !== prevWidth) {
+                popup.popper.update();
+            }
+        }
+    }, [popupRef, reference.getBoundingClientRect()]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Does this lint rule need to be disabled?

Reply:
No, I don't think so.